### ISSUE TITLE
test_nexus_openapi failing

### ISSUE THREAD

**Issue by andrewjstone** [oxidecomputer/omicron#2706]
On one of my [runs](https://buildomat.eng.oxide.computer/wg/0/details/01GWQSNP4ZH6VHW3DA7ASB5NW6/ogf0ITfgiF6NiNAmFsGYhw0yi0J5g1r7jT7NhzN430b2vXHN/01GWQSPK0VQYRP5P12MRNP4QY6#S5227) for a wicket-dbg PR (which has totally unrelated code), I was seeing the following error:

```

225 | 2023-03-29T23:31:43.012Z | failures:
-- | -- | --
5226 | 2023-03-29T23:31:43.017Z |  
5227 | 2023-03-29T23:31:43.022Z | ---- integration_tests::commands::test_nexus_openapi stdout ----
5228 | 2023-03-29T23:31:43.027Z | writing temp config: /var/tmp/omicron_tmp/test_commands_config.60871.7
5229 | 2023-03-29T23:31:43.033Z | @@ -711,18 +711,20 @@
5230 | 2023-03-29T23:31:43.038Z | "operationId": "disk_bulk_write_import",
5231 | 2023-03-29T23:31:43.043Z | "parameters": [
5232 | 2023-03-29T23:31:43.048Z | {
5233 | 2023-03-29T23:31:43.053Z | "in": "path",
5234 | 2023-03-29T23:31:43.059Z | "name": "disk",
5235 | 2023-03-29T23:31:43.064Z | +            "description": "Name or ID of the disk",
5236 | 2023-03-29T23:31:43.069Z | "required": true,
5237 | 2023-03-29T23:31:43.074Z | "schema": {
5238 | 2023-03-29T23:31:43.080Z | "$ref": "#/components/schemas/NameOrId"
5239 | 2023-03-29T23:31:43.085Z | }
5240 | 2023-03-29T23:31:43.090Z | },
5241 | 2023-03-29T23:31:43.096Z | {
5242 | 2023-03-29T23:31:43.101Z | "in": "query",
5243 | 2023-03-29T23:31:43.106Z | "name": "project",
5244 | 2023-03-29T23:31:43.111Z | +            "description": "Name or ID of the project",
5245 | 2023-03-29T23:31:43.117Z | "schema": {
5246 | 2023-03-29T23:31:43.122Z | "$ref": "#/components/schemas/NameOrId"
5247 | 2023-03-29T23:31:43.127Z | }
5248 | 2023-03-29T23:31:43.132Z | }
5249 | 2023-03-29T23:31:43.138Z | ],
5250 | 2023-03-29T23:31:43.143Z | @@ -758,18 +760,20 @@
5251 | 2023-03-29T23:31:43.148Z | "operationId": "disk_bulk_write_import_start",
5252 | 2023-03-29T23:31:43.233Z | "parameters": [
5253 | 2023-03-29T23:31:43.239Z | {
5254 | 2023-03-29T23:31:43.244Z | "in": "path",
5255 | 2023-03-29T23:31:43.249Z | "name": "disk",
5256 | 2023-03-29T23:31:43.254Z | +            "description": "Name or ID of the disk",
5257 | 2023-03-29T23:31:43.260Z | "required": true,
5258 | 2023-03-29T23:31:43.266Z | "schema": {
5259 | 2023-03-29T23:31:43.271Z | "$ref": "#/components/schemas/NameOrId"
5260 | 2023-03-29T23:31:43.276Z | }
5261 | 2023-03-29T23:31:43.281Z | },
5262 | 2023-03-29T23:31:43.293Z | {
5263 | 2023-03-29T23:31:43.298Z | "in": "query",
5264 | 2023-03-29T23:31:43.304Z | "name": "project",
5265 | 2023-03-29T23:31:43.309Z | +            "description": "Name or ID of the project",
5266 | 2023-03-29T23:31:43.314Z | "schema": {
5267 | 2023-03-29T23:31:43.320Z | "$ref": "#/components/schemas/NameOrId"
5268 | 2023-03-29T23:31:43.325Z | }
5269 | 2023-03-29T23:31:43.331Z | }
5270 | 2023-03-29T23:31:43.336Z | ],
5271 | 2023-03-29T23:31:43.341Z | @@ -795,18 +799,20 @@
5272 | 2023-03-29T23:31:43.346Z | "operationId": "disk_bulk_write_import_stop",
5273 | 2023-03-29T23:31:43.351Z | "parameters": [
5274 | 2023-03-29T23:31:43.356Z | {
5275 | 2023-03-29T23:31:43.361Z | "in": "path",
5276 | 2023-03-29T23:31:43.366Z | "name": "disk",
5277 | 2023-03-29T23:31:43.371Z | +            "description": "Name or ID of the disk",
5278 | 2023-03-29T23:31:43.377Z | "required": true,
5279 | 2023-03-29T23:31:43.382Z | "schema": {
5280 | 2023-03-29T23:31:43.387Z | "$ref": "#/components/schemas/NameOrId"
5281 | 2023-03-29T23:31:43.392Z | }
5282 | 2023-03-29T23:31:43.398Z | },
5283 | 2023-03-29T23:31:43.403Z | {
5284 | 2023-03-29T23:31:43.408Z | "in": "query",
5285 | 2023-03-29T23:31:43.413Z | "name": "project",
5286 | 2023-03-29T23:31:43.418Z | +            "description": "Name or ID of the project",
5287 | 2023-03-29T23:31:43.423Z | "schema": {
5288 | 2023-03-29T23:31:43.428Z | "$ref": "#/components/schemas/NameOrId"
5289 | 2023-03-29T23:31:43.434Z | }
5290 | 2023-03-29T23:31:43.439Z | }
5291 | 2023-03-29T23:31:43.445Z | ],
5292 | 2023-03-29T23:31:43.450Z | @@ -832,18 +838,20 @@
5293 | 2023-03-29T23:31:43.455Z | "operationId": "disk_finalize_import",
5294 | 2023-03-29T23:31:43.460Z | "parameters": [
5295 | 2023-03-29T23:31:43.466Z | {
5296 | 2023-03-29T23:31:43.471Z | "in": "path",
5297 | 2023-03-29T23:31:43.476Z | "name": "disk",
5298 | 2023-03-29T23:31:43.482Z | +            "description": "Name or ID of the disk",
5299 | 2023-03-29T23:31:43.487Z | "required": true,
5300 | 2023-03-29T23:31:43.492Z | "schema": {
5301 | 2023-03-29T23:31:43.497Z | "$ref": "#/components/schemas/NameOrId"
5302 | 2023-03-29T23:31:43.502Z | }
5303 | 2023-03-29T23:31:43.507Z | },
5304 | 2023-03-29T23:31:43.513Z | {
5305 | 2023-03-29T23:31:43.518Z | "in": "query",
5306 | 2023-03-29T23:31:43.523Z | "name": "project",
5307 | 2023-03-29T23:31:43.528Z | +            "description": "Name or ID of the project",
5308 | 2023-03-29T23:31:43.534Z | "schema": {
5309 | 2023-03-29T23:31:43.539Z | "$ref": "#/components/schemas/NameOrId"
5310 | 2023-03-29T23:31:43.544Z | }
5311 | 2023-03-29T23:31:43.550Z | },
5312 | 2023-03-29T23:31:43.555Z | {
5313 | 2023-03-29T23:31:43.561Z | @@ -878,18 +886,20 @@
5314 | 2023-03-29T23:31:43.566Z | "operationId": "disk_import_blocks_from_url",
5315 | 2023-03-29T23:31:43.571Z | "parameters": [
5316 | 2023-03-29T23:31:43.583Z | {
5317 | 2023-03-29T23:31:43.588Z | "in": "path",
5318 | 2023-03-29T23:31:43.593Z | "name": "disk",
5319 | 2023-03-29T23:31:43.598Z | +            "description": "Name or ID of the disk",
5320 | 2023-03-29T23:31:43.603Z | "required": true,
5321 | 2023-03-29T23:31:43.608Z | "schema": {
5322 | 2023-03-29T23:31:43.613Z | "$ref": "#/components/schemas/NameOrId"
5323 | 2023-03-29T23:31:43.618Z | }
5324 | 2023-03-29T23:31:43.623Z | },
5325 | 2023-03-29T23:31:43.629Z | {
5326 | 2023-03-29T23:31:43.634Z | "in": "query",
5327 | 2023-03-29T23:31:43.639Z | "name": "project",
5328 | 2023-03-29T23:31:43.644Z | +            "description": "Name or ID of the project",
5329 | 2023-03-29T23:31:43.660Z | "schema": {
5330 | 2023-03-29T23:31:43.668Z | "$ref": "#/components/schemas/NameOrId"
5331 | 2023-03-29T23:31:43.676Z | }
5332 | 2023-03-29T23:31:43.681Z | }
5333 | 2023-03-29T23:31:43.692Z | ],
5334 | 2023-03-29T23:31:43.731Z |  
5335 | 2023-03-29T23:31:43.766Z | thread 'integration_tests::commands::test_nexus_openapi' panicked at 'assertion failed: string doesn't match the contents of file: "../openapi/nexus.json" see diffset above
5336 | 2023-03-29T23:31:43.785Z | set EXPECTORATE=overwrite if these changes are intentional', nexus/tests/integration_tests/commands.rs:172:5
5337 | 2023-03-29T23:31:43.800Z | note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by david-crespo** [comment]
I noticed this on a recent PR. This is due to #2690 and something being added on main that wasn’t covered. We just need to update the spec on main.

**Comment by andrewjstone** [comment]
Fixed by #2707
